it/connectors/VatSubscriptionConnectorISpec.scala

```diff
@@ -48,7 +49,8 @@ class VatSubscriptionConnectorISpec extends IntegrationBaseSpec {
     firstName = Some("Dave"),
     lastName = Some("Taylor"),
     organisationName = Some("D Taylor's Cars"),
-    tradingName = Some("DT Autos")
+    tradingName = Some("DT Autos"),
+    contactPreference = None
```

Comment:
Could you change this so that `contactPreference` is present and update the relevant JSON in `VatSubscriptionStub` to also populate it. It doesn't change the outcome of this test but it's nice to have an additional check that every part of the JSON reads is working, and is consistent with the other fields in this val.